plugins/calibre.koplugin/metadata.lua

```diff
@@ -121,11 +121,7 @@ function CalibreMetadata:loadBookList()
         return {}
     end
     local books, err
-    if attr.size > MAX_JSON_FILESIZE then
```

Comment:
PS For future reference, while I expressed that I'm not sure if this should be removed, the intent behind this PR is incomplete. Since this line should then also be removed:

https://github.com/koreader/koreader/blob/eaac8c57fdae2a7ed74154663652eefe247df3a3/plugins/calibre.koplugin/metadata.lua#L55-L57

Reply:
true fair enough, lemme poke with it some more